src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -596,7 +596,6 @@ export default class DefaultAudioVideoController
     this.meetingSessionContext.connectionMonitor = new SignalingAndMetricsConnectionMonitor(
       this,
       this._realtimeController,
-      this._videoTileController,
```

Comment:
For understanding, why do we need to remove this?

Reply:
We don't use it anymore, it was being used to send videoSendHealthDidChange only when video is on.